index.rst

```diff
@@ -268,7 +268,7 @@ are some tips:
 -  Check out the lightning talks, and get excited about presenting one
    at next year's conference.
 
-**Sample strategy for a second or higher Write the Docs conference**
+**Sample strategy for a second or higher conference**
```

Comment:
removed conf name to make it less confusing, since there will only be one PyCascades conf